apps/web/src/pages/Communities/CommunityLayout.tsx

```diff
@@ -0,0 +1,151 @@
+import { useIntl } from "react-intl";
+import { Outlet } from "react-router-dom";
+import ClipboardDocumentListIcon from "@heroicons/react/24/outline/ClipboardDocumentListIcon";
+import { useQuery } from "urql";
+
+import { ThrowNotFound, Pending } from "@gc-digital-talent/ui";
+import { getLocalizedName } from "@gc-digital-talent/i18n";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { ROLE_NAME } from "@gc-digital-talent/auth";
+
+import SEO from "~/components/SEO/SEO";
+import AdminHero from "~/components/Hero/AdminHero";
+import useRoutes from "~/hooks/useRoutes";
+import useCurrentPage from "~/hooks/useCurrentPage";
+import useRequiredParams from "~/hooks/useRequiredParams";
+import { PageNavInfo } from "~/types/pages";
+
+import RequireAuth from "../../components/RequireAuth/RequireAuth";
```

Comment:
This can be updated to an absolure path: `~/components`

Reply:
5ed343c48da3fe8568f7dedf7ace482ab7343c72